debug/src/debug.js

```diff
@@ -198,9 +198,11 @@ export function initDebug() {
 	};
 
 	options.diffed = (vnode) => {
+		if (oldDiffed) oldDiffed(vnode);
+
 		if (vnode._component && vnode._component.__hooks) {
 			let hooks = vnode._component.__hooks;
-			hooks._list.forEach(hook => {
+			(hooks._list || []).forEach(hook => {
```

Comment:
Default anyway if there are for example two or more positions where optionss.diffed is used.